### ISSUE TITLE
model_name or network

### ISSUE THREAD

**Issue by h030162** [toandaominh1997/EfficientDet.Pytorch#28]
python train.py --dataset_root /root/data/VOCdevkit/ --model_name effcientdet-d0 # Example

not model_name but network

**Comment by toandaominh1997** [comment]
Yes, i replaced --model_name to --network.I changed in README.md 
Thanks @h030162
